lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -21,6 +21,43 @@ def self.email(params = {}, _secrets = {}, _context = {})
 
         private_class_method def self.email_cleanup(*)
         end
+
+        def self.provision_execute(_params = {}, _secrets = {}, context = {})
+          return unless context.execution["_object_type"] == "ServiceTemplateProvisionTask"
+
+          request_task_id = context.execution["_object_id"]
+          ::MiqRequestTask.find(request_task_id).execute
+
+          {"miq_request_task_id" => request_task_id}
+        end
+
+        private_class_method def self.provision_execute_status!(runner_context)
+          request_task = ::MiqRequestTask.find_by(:id => runner_context["miq_request_task_id"])
+          return if request_task.nil?
+
+          case request_task.statemachine_task_status
+          when "error"
+            reason = task.message
+            reason = reason[7..] if reason[0..6] == "Error: "
```

Comment:
```suggestion
            reason = reason[7..] if reason.start_with?("Error: ")
```

or, perhaps just use sub which I think is more readable

```suggestion
            reason = reason.sub(/^Error: ", "")
```